taskfiles/Proto.yml

```diff
@@ -0,0 +1,11 @@
+version: '3'
+
+tasks:
+  # Update gRPC protocol definitions
+  protogen:
+    run: once
+    dir: proto
+    cmds:
+      - protoc --go_out=plugins=grpc:. bmc.proto
+      - protoc --go_out=plugins=grpc:. config.proto
+      
```

Comment:
Remove the newline at end of file.

Reply:
I think I was generally a bit inconsistent with newlines at the end of the Taskfiles, I'll clean that up later today then.

Reply:
My perception was wrong. Somehow the complaint was "No newline at end of file" and I interpreted that as "remove the line" Seems like it was complaining about the missing of it.